client/my-sites/plugins/plugins-list/style.scss

```diff
@@ -1,56 +1,25 @@
-.plugins-list {
-	margin-bottom: 16px;
-}
-
-.plugins-list .plugin-item {
-	@include breakpoint-deprecated( "<1040px" ) {
-		width: 100%;
-
-		&:nth-last-child(n + 2) {
-			border-bottom-width: 1px;
-		}
-	}
-
-	@include breakpoint-deprecated( ">1040px" ) {
-		flex-direction: column;
-		width: 33.33%;
-		border-right-width: 1px;
-
-		&:nth-last-child(n + 4) {
-			border-bottom-width: 1px;
-		}
-
-		// Provider proper bottom borders in the second-last row when the number
-		// of items is not multiple of 3.
-		&:nth-child(3n) {
-			border-right-width: 0;
-			&:nth-last-child(3),
-			&:nth-last-child(2) {
-				border-bottom-width: 1px;
-			}
-		}
-
-		&:nth-child(3n - 1):nth-last-child(3) {
-			border-bottom-width: 1px;
-		}
-
-		.plugin-item__count,
-		.plugin-item__actions {
-			align-self: flex-end;
-			flex-direction: column;
-			margin-top: -6px;
-			padding-top: 0;
-			text-align: right;
-		}
+body.is-section-plugins .plugin-management-wrapper {
 
-		.plugin-item__count {
-			flex-direction: row;
-		}
+	.navigation-header {
+		border-block-end: 1px solid var(--color-border-secondary);
+		padding-inline: 48px;
 	}
 }
 
-.plugins-list__elements {
-	display: flex;
-	flex-flow: row wrap;
-	padding: 0;
+body.is-section-plugins .plugin-management-wrapper,
+.is-section-jetpack-cloud-plugin-management {
+    --wp-components-color-accent: var(--studio-automattic-blue-50);
+
+    .dataviews-wrapper .dataviews-view-table {
+        .dataviews-view-table__row {
+            .plugin-icon {
+                max-width: 35px;
+                max-height: 35px;
```

Comment:
We could remove this CSS, but the icon would have 56px, which is too big for this screen. It would come [from this file](https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/plugins/plugin-icon/style.scss#L8-L9).

Reply:
Can we have a separate PR to make this a prop of the `PluginIcon` component, and use that instead of style overrides?